### ISSUE TITLE
Installation not detected

### ISSUE THREAD

**Issue by rauscheb** [UncleGoogle/galaxy-integration-osu#5]
My installation of osu is not being detected in GoG. It was installed before I had the plugin and that wasn't detected. I uninstalled and re-installed using the plugin, but it's still not detected. Restarted GoG and my computer. Nothing.

**Comment by UncleGoogle** [comment]
Hi, where have you installed osu? I mean directory path to application. Do you use latest plugin version?

**Comment by UncleGoogle** [comment]
Aattach logs please

- go to logs location: %programdata%\GOG.com\Galaxy\logs
- look for file with 'egg' in name
- archive logs to zip file and attach here

**Comment by rauscheb** [comment]
When I wrote this, I thought I was on the latest, but it looks like v1.0.1 doesn't have a compiled version on the releases page. It looks like I was using 1.0.0 because of that.

I used the commands on the README to compile 1.0.1 myself, but it's not copying over the source to my plugins folder, just an "osu" file with version history inside it.


[plugin-egg-79764115-b455-4ff0-be13-a316c7c53dec.log](https://github.com/UncleGoogle/galaxy-integration-osu/files/4913715/plugin-egg-79764115-b455-4ff0-be13-a316c7c53dec.log)

![image](https://user-images.githubusercontent.com/17773362/87328182-b83b8d00-c4fa-11ea-84a9-d58681a78efa.png)
